src/main/java/at/hannibal2/skyhanni/utils/LorenzUtils.kt

```diff
@@ -297,7 +294,7 @@ object LorenzUtils {
     fun isControlKeyDown() = OSUtils.isKeyHeld(Keyboard.KEY_LCONTROL) || OSUtils.isKeyHeld(Keyboard.KEY_RCONTROL)
 
     // A mac-only key, represents Windows key on windows (but different key code)
-    fun isCommandKeyDown() = OSUtils.isKeyHeld(Keyboard.KEY_LMETA) || OSUtils.isKeyHeld(Keyboard.KEY_LMETA)
```

Comment:
How have you found this error? :O

Reply:
Linting software highlighted it as a duplicate check!